--- a/seiyria-bootstrap-slider/js/bootstrap-slider.js	2015-09-28 00:53:26.000000000 +0100
+++ b/seiyria-bootstrap-slider/js/bootstrap-slider.js	2015-09-28 01:08:10.000000000 +0100
@@ -986,6 +986,10 @@
 						var percentage = this.options.ticks_positions[i] ||
 							100 * (this.options.ticks[i] - minTickValue) / (maxTickValue - minTickValue);
 
+						if (this.options.reversed) {
+							percentage = 100 - percentage;
+						}
+
 						this.ticks[i].style[this.stylePos] = percentage + '%';
 
 						/* Set class labels to denote whether ticks are in the selection */
@@ -1005,7 +1009,7 @@
 
 							if (this.options.ticks_positions[i] !== undefined) {
 								this.tickLabels[i].style.position = 'absolute';
-								this.tickLabels[i].style[this.stylePos] = this.options.ticks_positions[i] + '%';
+								this.tickLabels[i].style[this.stylePos] = percentage + '%';
 								this.tickLabels[i].style[styleMargin] = -labelSize/2 + 'px';
 							}
 						}
